src/Raven.Studio/wwwroot/App/views/database/tasks/ongoingTasks.html

```diff
@@ -336,10 +336,71 @@ <h5 class="tasks-list-item subscription no-text-transform"><i class="icon-subscr
                                             </div>
                                             <!-- /ko -->
                                             <div class="property-item">
-                                                <div class="property-name">Change vector for next batch: </div>
-                                                <div class="property-value text-details" data-bind="text: changeVectorForNextBatchStartingPoint() || 'N/A'"></div>
+                                                <div class="property-name">Next batch starting point: </div>
+                                                <div class="property-value text-details">
+                                                    <span>
+                                                        <span data-bind="foreach: changeVectorForNextBatchStartingPointFormatted, visible: changeVectorForNextBatchStartingPoint()">
+                                                                <span class="badge badge-default" data-bind="text: shortFormat"></span>
+                                                        </span>
+                                                        <span>
+                                                            <small data-bind="visible: !nextBatchStartingPointDocumentFound()">
+                                                                <a href="#" data-bind="click: _.partial(tryGetDocumentIDFromChangeVector, 'NextBatchStartingPoint'), 'NextBatchStartingPoint'), visible: !spinners.documentIDForNextBatchStartingPointLoading()" 
+                                                                   title="Search for the document that matches this change-vector">
+                                                                    <i class="icon-search"></i>
+                                                                </a>
+                                                                <span data-bind="visible: spinners.documentIDForNextBatchStartingPointLoading">
+                                                                    &nbsp;&nbsp;&nbsp;<i class="btn-spinner"></i>
```

Comment:
nbsp nbsp

Reply:
removed one 'nbsp' but left only one 'nbsp' as it was mandatory